plugin/fulltext/mysql-test/ngram/ngram_token_size.test

```diff
@@ -0,0 +1,82 @@
+--echo # Check 'ngram_token_size' variable.
+
+--echo #
+--echo # It's a global variable.
+SELECT @@global.ngram_token_size;
+SHOW GLOBAL VARIABLES LIKE 'ngram_token_size';
+SELECT * FROM information_schema.global_variables WHERE variable_name='ngram_token_size';
+
+--echo #
+--echo # Not a session variable.
+--error ER_INCORRECT_GLOBAL_LOCAL_VAR
+SELECT @@session.ngram_token_size;
+SHOW SESSION VARIABLES LIKE 'ngram_token_size';
+SELECT * FROM information_schema.session_variables WHERE variable_name='ngram_token_size';
+
+--let $saved_ngram_token_size= `SELECT @@global.ngram_token_size`
+
+--echo #
+--echo # 0 is outside allowed range [1, 10], so it's changed to 1.
+SET GLOBAL ngram_token_size= 0;
+SELECT * FROM information_schema.global_variables WHERE variable_name='ngram_token_size';
+
+--echo #
+--echo # 10 is an acceptable value.
+SET GLOBAL ngram_token_size= 10;
+SELECT * FROM information_schema.global_variables WHERE variable_name='ngram_token_size';
+
+--echo #
+--echo # 1 is an acceptable value.
+SET GLOBAL ngram_token_size= 1;
+SELECT * FROM information_schema.global_variables WHERE variable_name='ngram_token_size';
+
+--echo #
+--echo # 11 is outside allowed range [1, 10], so it's changed to 10.
+SET GLOBAL ngram_token_size= 11;
+SELECT * FROM information_schema.global_variables WHERE variable_name='ngram_token_size';
+
+--echo #
+--echo # Try 3-grams.
+SET GLOBAL ngram_token_size= 3;
+CREATE TABLE t (str TEXT, FULLTEXT INDEX (str) WITH PARSER ngram) Engine=InnoDB;
```

Comment:
Before this change, could we create and populate a table with a different `ngram_token_size`, and then query and update that table while `ngram_token_size=3` is in effect?

Reply:
Is that to have index that have both 2-grams and 3-grams simultaneously? Added similar case to the test a dozen of lines lower, after switching `ngram_token_size` to 2.